collections/src/main/scala/strawman/collection/immutable/LazyList.scala

```diff
@@ -506,7 +508,8 @@ object LazyList extends LazyListFactory[LazyList] {
   }
 
   object #:: {
-    def unapply[A](s: LazyList[A]): Evaluated[A] = s.force
+    def unapply[A](s: LazyList[A]): Option[(A, LazyList[A])] =
+      if (s.nonEmpty) Some(s.head, s.tail) else None
```

Comment:
You should add extra parens to make it clear that `Some` takes one parameter (which is a pair)

Reply:
Fixed.